widget/ui/src/components/Header/Header.tsx

```diff
@@ -5,24 +5,46 @@ import React from 'react';
 
 import { Typography } from '../Typography';
 
-import { Container, globalHeaderStyles, Suffix } from './Header.styles';
+import { Container, globalHeaderStyles, Prefix, Suffix } from './Header.styles';
 
 export function Header({
   prefix,
   suffix,
   title,
+  disableCurves,
+  titlePosition,
+  transparent,
+  css,
 }: PropsWithChildren<PropTypes>) {
   globalHeaderStyles();
 
+  const renderTitle = () => {
+    if (!title) {
+      return null;
+    } else if (typeof title === 'string') {
+      return (
+        <Typography variant="headline" size="small">
+          {title}
+        </Typography>
+      );
+    }
+    return title;
+  };
+
   return (
-    <Container>
-      {prefix}
-      <Typography variant="headline" size="small">
-        {title}
-      </Typography>
+    <Container
+      titlePosition={titlePosition || 'center'}
+      transparent={!!transparent}
+      css={css}>
+      <Prefix>{prefix}</Prefix>
+      {renderTitle()}
       <Suffix>{suffix}</Suffix>
-      <div className="rng-curve-left"></div>
-      <div className="rng-curve-right"></div>
+      {!disableCurves && (
```

Comment:
You may ask what happens if I want to show a centered title without any thing in left and right:

```jsx
<Header title={...} prefix={<></>} suffix={<></>}  />
```

Or title in left, something on right:
```jsx
<Header title={...} suffix={...}  />
```

Reply:
Hope you got the idea. feel free to ask any questions if you have any.

Reply:
The hole point of this PR is using `absolute` style for header items instead of `space-between` because of the problems like changing the position of title in header in result of changing the width of suffix and prefix.

Reply:
I see. I'm good with your new change. Although, home page is broken, rest of the pages looks good.

Reply:
I don't remember why we added the corner to `Header` at the first place, the only thing on top off my head is it has a condition on scrolling.  you can check it out later, we can skip this for now to keep the scope of this PR.

Reply:
Resolved homepage bug by setting prefix as `null` instead of `<></>` if a prefix is not available.